### PR TITLE
cmc hotfixes

### PR DESCRIPTION
**Pull request by PaulRitter**
no more magic indexes

no more one line ifs

ghosts can now use it

now using DEAD define to check if person is indeed dead

### CODE REVIEW

File: code/modules/cmc/crew.dm [hunk 1]
```diff
@@ -530,15 +541,15 @@ Crew Monitor by Paul, based on the holomaps by Deity
...
-		var/ijob = entry[9]
+		var/see_x = entry[ENTRY_SEE_X]
+		var/see_y = entry[ENTRY_SEE_Y]
+		var/mob/living/carbon/H = entry[ENTRY_MOB]
+		var/name = entry[ENTRY_NAME]
+		var/job = entry[ENTRY_ASSIGNMENT]
+		var/stat = entry[ENTRY_STAT]
+		var/list/damage = entry[ENTRY_DAMAGE]
+		var/player_area = entry[ENTRY_AREA]
+		var/ijob = entry[ENTRY_POS]
```
Comment (ShiftyRail) [inline code/modules/cmc/crew.dm]: `ijob` `ENTRY_POS`
:thinking:

File: code/modules/cmc/crew.dm [hunk 2]
```diff
@@ -400,8 +410,9 @@ Crew Monitor by Paul, based on the holomaps by Deity
 
 		for(var/entry in entries[holomap_z[uid]])
 			//can only be our z, so i'm not checking that, only if we have a pos
-			if(entry[1]) addCrewMarker(user, entry[1], entry[2], entry[3], entry[4], entry[5], entry[6], entry[7], entry[8], entry[10])
-
+			if(entry[1])
```
Comment (ShiftyRail) [inline code/modules/cmc/crew.dm]: Is this about weather or not the list is here at all? 
Also make it `entry[ENTRY_SEE_X]`

Comment (PaulRitter) [reply code/modules/cmc/crew.dm]: its supposed to check if we got a pos so we should make a crewmaker. altough that is also checked in addcrewmarker

**Comment by PaulRitter** [comment]
~actually dont merge, i'll add ghost usability into this pr~

ready to merge

**Pull request merged**